source/js/inc/app.js

```diff
@@ -33,7 +33,7 @@ $(document).ready(function () {
 						min = min === null ? amount : min; // in case of null, set to amount so it will pass
 						max = max === null ? amount : max; // in case of null, set to amount so it will pass
 
-						return validFormat && amount >= min && amount <= max;
+						return validFormat && (amount >= min) && (amount <= max);
```

Comment:
@jedlin-kiva speaking with @emuvente 
We should should add these parens around the return value checks in oridinalDayOfMonth too...